.circleci/config.yml

```diff
@@ -123,6 +127,22 @@ workflows:
               ignore: /.*/
 
 jobs:
+  lint:
+    docker:
+      - image: goswagger/builder:debian
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - go-mod-v1-{{ checksum "go.sum" }}
+      - run:
+          name: Install golangci-lint
+          command: go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.24.0
```

Comment:
can you make the tag: `@latest`

Reply:
I have pinned this on purpose. Lately, on some other repos of mine, more recent versions were broken.
Let me push some more commits downgrading from latest up to the most recent version that works.

Reply:
if we don't specify `-u` and use `GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint@latest`
There should be no issue because it will just resolve to the highest tag version.

Reply:
With go1.14, "on" should be the default. Is there still a need to enforce GO111MODULE="on" nowadays?
Thinking about the impact of "-u"...

Reply:
correct on the env var.
the impact of `-u` is that it ignores the content of the go.mod file of the tagged version and updates the dependencies of that module. So that's when things break because they are not using the correct versions.